Realm/RLMObject.h

```diff
@@ -307,7 +309,7 @@
  @return    An object of the subclass type or nil if an object with the given primary key does not exist.
  @see       -primaryKey
  */
-+ (instancetype)objectForPrimaryKey:(id)primaryKey;
++ (nullable instancetype)objectForPrimaryKey:(id)primaryKey;
```

Comment:
I assume exactly one object would be allowed to have a null value for the primary key? I'm struggling to think of a scenario in which that would be useful.